.github/workflows/reusable_pre_release.yaml

```diff
@@ -139,6 +139,10 @@ jobs:
           # The upload script expects this to authenticate.
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: make ci/prerelease
+      - uses: newrelic/coreint-automation/test-fips-action@v3
+        if: ${{ inputs.upload_fips_packages }}
+        with:
+          binaries_path: "./dist"
```

Comment:
As I can see on the script below this will only execute it on the fips binaries, correct?

Reply:
Yes, as the binary name is same for both `fips` and `non-fips`, we differentiate based on the folder name and only test the binaries that are inside a folder with `fips` in the name.
In below example, it will test for `./dist/nri-nix-fips_linux_amd64_v1/nri-redis` and `./dist/nri-nix-fips_linux_arm64_v8.0/nri-redis`
```sh
├── artifacts.json
├── config.yaml
├── metadata.json
├── nri-nix-fips_linux_amd64_v1
│   └── nri-redis
├── nri-nix-fips_linux_arm64_v8.0
│   └── nri-redis
├── nri-nix_darwin_amd64_v1
│   └── nri-redis
├── nri-nix_darwin_arm64_v8.0
│   └── nri-redis
├── nri-nix_linux_386_sse2
│   └── nri-redis
├── nri-nix_linux_amd64_v1
│   └── nri-redis
├── nri-nix_linux_arm64_v8.0
│   └── nri-redis
├── nri-nix_linux_arm_6
│   └── nri-redis
├── nri-redis-1.12.1-SNAPSHOT-9b4be8c-1.386.rpm
├── nri-redis-1.12.1-SNAPSHOT-9b4be8c-1.arm.rpm
├── nri-redis-1.12.1-SNAPSHOT-9b4be8c-1.arm64.rpm
├── nri-redis-1.12.1-SNAPSHOT-9b4be8c-1.x86_64.rpm
├── nri-redis-fips-1.12.1-SNAPSHOT-9b4be8c-1.arm64.rpm
├── nri-redis-fips-1.12.1-SNAPSHOT-9b4be8c-1.x86_64.rpm
├── nri-redis-fips-fips_1.12.1-SNAPSHOT-9b4be8c-1_amd64.deb
├── nri-redis-fips-fips_1.12.1-SNAPSHOT-9b4be8c-1_arm64.deb
├── nri-redis-fips_linux_1.12.1-SNAPSHOT-9b4be8c_amd64_dirty.tar.gz
├── nri-redis-fips_linux_1.12.1-SNAPSHOT-9b4be8c_arm64_dirty.tar.gz
├── nri-redis_1.12.1-SNAPSHOT-9b4be8c-1_386.deb
├── nri-redis_1.12.1-SNAPSHOT-9b4be8c-1_amd64.deb
├── nri-redis_1.12.1-SNAPSHOT-9b4be8c-1_arm.deb
├── nri-redis_1.12.1-SNAPSHOT-9b4be8c-1_arm64.deb
├── nri-redis_1.12.1-SNAPSHOT-9b4be8c_checksums.txt
├── nri-redis_darwin_1.12.1-SNAPSHOT-9b4be8c_amd64_dirty.tar.gz
├── nri-redis_darwin_1.12.1-SNAPSHOT-9b4be8c_arm64_dirty.tar.gz
├── nri-redis_linux_1.12.1-SNAPSHOT-9b4be8c_386_dirty.tar.gz
├── nri-redis_linux_1.12.1-SNAPSHOT-9b4be8c_amd64_dirty.tar.gz
├── nri-redis_linux_1.12.1-SNAPSHOT-9b4be8c_arm64_dirty.tar.gz
└── nri-redis_linux_1.12.1-SNAPSHOT-9b4be8c_arm_dirty.tar.gz
```